BlackNectar/BlackNectar/SideMenuFilterViewController.swift

```diff
@@ -83,19 +88,19 @@ class SideMenuFilterViewController: UITableViewController {
 
         if isRestaurant == false {
```

Comment:
It's a little weird to compare boolean values to true or false. It is much more natural to just say:
```swift
if isRestaurant {
}
```